### DISCUSSION TITLE
Passing array as a variable

### DISCUSSION THREAD

**Discussion by lindseyschuler** [graphql-dotnet/graphql-client#585] [General]
Hi,

I'm trying to pass an array in as a variable, I've tested it in a graphql query tool and it works fine, but when I try it through the GraphQl client I get a bad request error
GraphQL.Client.Http.GraphQLHttpRequestException: 'The HTTP request failed with status code BadRequest'
```

var query = new GraphQLRequest
			{
				Query = @"query ResortProductSetup($rposResortCodes: [Int!]) {
                  all_resort(where: {product_setup: {rpos_resort_code_in: $rposResortCodes}) {
                    " + QueryFragmentConstants.ResortProductSetupQuery + @"
                  }
                }",
				OperationName = "ResortProductSetup",
                Variables = new { rposResortCodes = resortCodes }
			};

```
resortCodes is an int[] that looks like this [1,2,3]

Is there something different I need to do in order to pass in an array

**Comment by rose-a** [comment]
Try passing it as a `List<int>` instead of an array...
